wheatley/bot.py

```diff
@@ -270,8 +285,8 @@ def _on_go(self) -> None:
             # because the rows on which these calls should have been called have already passed.
             # Therefore, we simply get them out as quickly as possible so they have the best chance
             # of being heard.
-            for (_index, _calls) in early_calls:
-                self._make_calls(_calls)
+            for (_index, calls_) in early_calls:  # `calls_` won't conflict with `wheatley.calls`
+                self._make_calls(calls_)
```

Comment:
I'm not sure `calls_` is better, it's inconsistent with `_index` (which is unused.)
Elsewhere we have used single letters, perhaps
```suggestion
            for (_, c) in early_calls: 
                self._make_calls(c)
```

Reply:
Yep I like this more.  Pity Python treats `_` as a legit variable name, but we don't read from it so it's fine.